extensions/civetweb/processors/ListenHTTP.cpp

```diff
@@ -182,21 +182,12 @@ void ListenHTTP::onSchedule(core::ProcessContext *context, core::ProcessSessionF
       options.emplace_back("yes");
     }
 
-    if (sslMinVer == "SSL2") {
-      options.emplace_back("ssl_protocol_version");
-      options.emplace_back(std::to_string(0));
-    } else if (sslMinVer == "SSL3") {
-      options.emplace_back("ssl_protocol_version");
-      options.emplace_back(std::to_string(1));
-    } else if (sslMinVer == "TLS1.0") {
-      options.emplace_back("ssl_protocol_version");
-      options.emplace_back(std::to_string(2));
-    } else if (sslMinVer == "TLS1.1") {
-      options.emplace_back("ssl_protocol_version");
-      options.emplace_back(std::to_string(3));
-    } else {
+    if (sslMinVer == "TLS1.2") {
       options.emplace_back("ssl_protocol_version");
       options.emplace_back(std::to_string(4));
+    } else {
+      logger_->log_error("Invalid SSL Minimum Version specified!");
```

Comment:
I think if we throw an exception here we have a chance of catching this earlier in the re-deployment process.

Reply:
Fair point, I will change this to an exception.